docs/advanced-table-feature.md

```diff
@@ -0,0 +1,65 @@
+#### Partitioned table
+
+Currently, Gravitino supports the following partitioning strategies:
+
+| Partitioning strategy | Json                                                | Java                           | SQL syntax                 | Description                                                                                                                 |
+|-----------------------|-----------------------------------------------------|--------------------------------|----------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| Identity              | `{"strategy":"identity","fieldName":["score"]}`     | `Transforms.identity("score")` | `PARTITION BY score`       | Partition by a field or reference                                                                                           |
```

Comment:
What's score?

Reply:
That's the name of a table column. Perhaps I need to add some information about it.